index.html

```diff
@@ -0,0 +1,138 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Movielist</title>
+    <link rel="stylesheet" href="./StyleSheets/Navigation.css" />
+    <link rel="stylesheet" href="./StyleSheets/normalize.css" />
+  </head>
+  <body>
+    <div class="container">
+      <aside class="nav">
+        <figure class="logo">
+          <img class="LogoImg Desktop" src="./images/Logo.svg" alt="" />
+          <div class="LogoImg TabletAndMobile"></div>
+        </figure>
+
+        <nav class="menu">
+          <menu class="pages">
+            <div class="search menuPages">
+              <img src="./images/Search.svg" alt="" class="iconimg" />
```

Comment:
We should always have some `alt` text which is related to the actual image.
https://accessibility.psu.edu/images/alttext/